openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RequestProfileParams.java

```diff
@@ -36,6 +38,14 @@
     @JsonIgnore
     private boolean hasRestrictions = false;
 
+    @ApiModelProperty(name = PARAM_ALLOW_UNSUITABLE, value = "Specifies if unsuitable roads should be allowed to find routes - default false" +
```

Comment:
Should reword to "Specifies if ways that might not be suitable (e.g. unknown pedestrian usage) should be included in finding routes - default false"

Reply:
done